splice/static/html/partials/authoring.html

```diff
@@ -1,61 +1,159 @@
-<div class="container-fluid">
-    <div class="alert-growl">
-        <alert ng-repeat="alert in alerts" type="{{alert.type}}" close="closeAlert()">
-            <span ng-bind-html="alert.msg"></span>
-        </alert>
+<script type="text/ng-template" id="template/confirmation.html">
+    <div class="modal" tabindex="999" role="dialog">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" ng-click="$hide()">&times;</button>
+                    <h2 class="modal-title text-danger">Warning!</h2>
+                </div>
+                <div class="modal-body">
+                    <p>You are about to publish tiles to with parameters:</p>
+                    <table class="table table-condensed">
+                        <thead>
+                            <tr>
+                                <th>Channel</th>
+                                <th>Schedule For</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr>
+                                <td>{{ channelIndex[channelSelect.id].name }}</td>
+                                <td>{{ deployConfig.scheduled ? deployConfig.scheduled.toString() : "Now" }}</td>
+                            </tr>
+                            <tr ng-show="deployConfig.scheduled">
+                                <td></td>
+                                <td>{{ deployConfig.scheduled ? deployConfig.scheduled.toUTCString() : "" }}</td>
```

Comment:
Tested this out locally. It looks like in this line + line 21 above we get the date written twice in different formats if deployConfig.scheduled is true?